{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/settings.py

```diff
@@ -200,13 +253,11 @@ class Prod(Deployed):
     DEBUG = False
 
     DATABASES = {
-        'default': {
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'NAME': getenv('POSTGRES_USER', ''),
-            'USER': getenv('POSTGRES_USER', ''),
-            'PASSWORD': getenv('POSTGRES_PASSWORD', 'password'),
-            'HOST': getenv('POSTGRES_SERVICE_HOST', 'localhost'),
+        "default": {
+            "ENGINE": "django.db.backends.postgresql_psycopg2",
+            "NAME": get_env("POSTGRES_USER", ""),
+            "USER": get_env("POSTGRES_USER", ""),
+            "PASSWORD": get_env("POSTGRES_PASSWORD", "password"),
+            "HOST": get_env("POSTGRES_SERVICE_HOST", "localhost"),
```

Comment:
This should probably be in the base and just override the settings in Dev

Reply:
Will takle this one in the settings task.